mesheryctl/internal/cli/root/perf/view.go

```diff
@@ -0,0 +1,94 @@
+package perf
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var outFormatFlag string
+
+var viewCmd = &cobra.Command{
+	Use:     "view",
+	Short:   "view perf profile",
+	Long:    `See the configuration of your performance profile`,
+	Example: "mesheryctl perf view [ performance test profile name ]",
+	Args:    cobra.MaximumNArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
+		if err != nil {
+			return errors.Wrap(err, "error processing config")
+		}
+		proName := args[0]
+		var req *http.Request
+		url := mctlCfg.GetBaseMesheryURL() + "/api/user/performance/profiles"
```

Comment:
```suggestion
		url := mctlCfg.GetBaseMesheryURL() + "/api/user/performance/profiles/" + profileID
```
`view` command is to only fetch one profile's details.